save-common/src/commonMain/kotlin/org/cqfn/save/core/result/TestResult.kt

```diff
@@ -14,10 +14,14 @@ import kotlinx.serialization.UseSerializers
  * @property resources test resources that have been used in this test
  * @property status final status of the test
  * @property debugInfo additional info that can be set during execution. Might be absent.
+ * @property missing number of missing warnings
+ * @property match number of match warnings
  */
 @Serializable
 data class TestResult(
     val resources: Plugin.TestFiles,
     val status: TestStatus,
     val debugInfo: DebugInfo? = null,
+    val missing: Int? = null,
```

Comment:
I feel like such information pollutes this class a little. It's warn-plugin-specific, while `TestResult` is shared class for all plugins. It's too low-level for `TestResult`, imo. I'd rather see it in `debugInfo` or maybe even in a special subclass of `DebugInfo` specifically for warn-plugin

Reply:
Yes. Agree)

Reply:
Fixed